### ISSUE TITLE
[BUG]: [karavi-topology & karavi-topology]: Fails in Openshift

### ISSUE THREAD

**Issue by coulof** [dell/helm-charts#54]
**Describe the bug**
After installing Karavi on Openshift with: `helm install karavi-observability dell/karavi-observability -n vxflexos -f karavi.powerflex.values.yaml --render-subchart-notes --set-file karavi-topology.certificateFile=karavi-topology.crt --set-file karavi-topology.privateKeyFile=karavi-topology.key --set-file karavi-metrics-powerflex.otelCollector.certificateFile=otel-collector.crt --set-file karavi-metrics-powerflex.otelCollector.privateKeyFile=otel-collector.key` the components `otel-collector/nginx-proxy` and `karavi-topology` fails to start with errors like:
```
2020/12/01 08:47:10 [emerg] 1#1: bind() to 0.0.0.0:443 failed (13: Permission denied)
nginx: [emerg] bind() to 0.0.0.0:443 failed (13: Permission denied)
```

The problem is that Openshift runs containers as unprivileged by default and don't allow to run containers on default ports.

Please consider using the [unprivileged nginx image](https://hub.docker.com/r/nginxinc/nginx-unprivileged) and make the topology component listen on a non-default port.

**Version of Helm and Kubernetes**:
```
[root@floshift-bastion ~]# oc version
Client Version: 4.5.15
Server Version: 4.5.15
Kubernetes Version: v1.18.3+2fbd7c7
[root@floshift-bastion ~]# helm version
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /root/floshift/auth/kubeconfig
version.BuildInfo{Version:"v3.3.4+5.el8", GitCommit:"1e63a4770a20072ed9f574013c01cc6e59881e48", GitTreeState:"clean", GoVersion:"go1.13.15"}
```

**Which chart**:
karavi-topology & karavi-topology

**Comment by ioeicicdsvc** [comment]
link: 1735

**Comment by gallacher** [comment]
This issue is being addressed as part of issue #61

**Comment by ameer-jabbar** [comment]
Will be available in karavi-observability-0.2.0

https://github.com/dell/helm-charts/pull/60 (branch: karavi-observability-0.2.0])
https://github.com/dell/karavi-topology/pull/85 (branch: main)
https://github.com/dell/karavi-metrics-powerflex/pull/81 (branch: main)
https://github.com/dell/karavi/pull/23 (branch: main)
